### PR TITLE
Blog post about lesson infrastructure meeting

### PR DESCRIPTION
**Pull request by rgaiacs**
# Requests

- [ ] @k8hertweck could you review the blog post
- [x] @ChristinaLK could you check your email address
- [x] @tracykteal you probably want to review before merge and maybe republish at Data Carpentry website
- [x] I don't remember Mark's last name. :-(

### CODE REVIEW

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+- stay in contact with maintainers of lesson
+- stay in contact with staffs
+
+[Lesson template](https://github.com/swcarpentry/styles),
+[lesson documentation](https://github.com/swcarpentry/lesson-example)
+and
+[workshop template](https://github.com/swcarpentry/workshop-template)
+**will have a new home in 2018**.
+If you have questions or want to help with this migration,
+[Christina Koch](mailto:koch.christinal@gmail.com) is the person you should contact.
```
Comment (ChristinaLK) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: Use my other gmail: christinakconnect@gmail.com

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+---
+
+On 5 September 2017 at 14:00UTC+0,
+the Lesson Infrastructure Subcommittee had their 2017 September meeting.
+This post will cover the topics discussed and their resolutions.
+
+## Software Carpentry and Data Carpentry merge
+
+With the [upcoming merge](https://software-carpentry.org/blog/2017/09/merger.html),
+this subcommittee needs to start thinking about streamline the thinking process and disconect from the organisations that maintain the lessons.
```
Comment (ChristinaLK) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: streamline --> streamlining
disconect -> disconnect

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+The responsibilities for this subcommittee will stay unchanged:
+
+- maintain [lesson template](https://github.com/swcarpentry/styles)
+- maintain [lesson documentation](https://github.com/swcarpentry/lesson-example)
+- maintain [workshop template](https://github.com/swcarpentry/workshop-template)
+- overview of what features the lessons will continue to have
+- stay in contact with maintainers of lesson
+- stay in contact with staffs
```
Comment (ChristinaLK) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: staffs -> staff

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+The subcommittee will meet again in November to provide update on some of the topics covered by this post
+and discuss new requests from the community.
+
+## Acknowledgement
+
+Thanks to 
+Kate Hertweck,
+Maneesha Sane,
+Mark
```
Comment (ChristinaLK) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: Laufersweiler
:)

**Review by ChristinaLK — COMMENTED** [review]
A few spelling things, my email + Mark's last name.

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+[lesson documentation](https://github.com/swcarpentry/lesson-example)
+and
+[workshop template](https://github.com/swcarpentry/workshop-template)
+**will have a new home in 2018**.
+If you have questions or want to help with this migration,
+[Christina Koch](mailto:koch.christinal@gmail.com) is the person you should contact.
+
+During the migration,
+we will solve the divergences between
+[Software Carpentry workshop template](https://github.com/swcarpentry/workshop-template)
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: between -> between the

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+and
+[Data Carpentry workshop template](https://github.com/datacarpentry/workshop-template).
+If you have questions or want to help with this migration,
+[Maneesha Sane](mailto:maneesha@software-carpentry.org) is the person you should contact.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: contact should be Tracy, so 
[Maneesha Sane] -> [Tracy Teal](mailto:tkteal@datacarpentry.org)

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+## Keyboard key visual look
+
+To improve the reader experience of the lesson,
+we will make a visual diference of keyboard keys that need to be pressed by the reader.
+We expected to merge the new CSS and documentation in the next few weeks
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: expected to -> expect to

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+and that release 2018.6 of all the lessons complain with the new visual.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: and that release, 2018.6, will contain all of lessons with this new look.

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+we will solve the divergences between
+[Software Carpentry workshop template](https://github.com/swcarpentry/workshop-template)
+and
+[Data Carpentry workshop template](https://github.com/datacarpentry/workshop-template).
+If you have questions or want to help with this migration,
+[Maneesha Sane](mailto:maneesha@software-carpentry.org) is the person you should contact.
+
+## Keyboard key visual look
+
+To improve the reader experience of the lesson,
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: To improve the look and feel of the lesson and the learners' ability to use them

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+we will make a visual diference of keyboard keys that need to be pressed by the reader.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: we will make the keyboard keys that need to be pressed by the learner, look different from the other components of the text, so they are highlighted more effectively.

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+We expected to merge the new CSS and documentation in the next few weeks
+and that release 2018.6 of all the lessons complain with the new visual.
+More information about the new visual is available [at this pull request](https://github.com/swcarpentry/styles/pull/165).
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: about the new visual -> about this new feature

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+Thanks to [Brandon Curtis](https://github.com/brandoncurtis) to propose the new visual.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: to propose the new visual -> for proposing this idea.

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+
+## Jekyll/Liquid include for images/figures
+
+To improve the reader experience of the lesson by providing a more uniform render of images,
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: To improve readability by providing a more uniform image rendering,

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+we will pursuit the proposal on [GitHub issue styles#161](https://github.com/swcarpentry/styles/issues/161)
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: we will pursuit -> we will pursue

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+after we review [lessons unit test suite](https://github.com/swcarpentry/styles/blob/gh-pages/bin/lesson_check.py)
+and its use by a continuous integration platform.
+
+## Citing the templates
+
+If you want are reusing the lesson template
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: If you are using the lesson template

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+and you to credit us,
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: and you want to credit us,

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 17]
```diff
@@ -0,0 +1,128 @@
...
+please use [Software Carpentry: Example Lesson at Zenodo](https://zenodo.org/record/838778#.WbPw1HVifCl).
+
+## Lesson release and hosting scheme
+
+[For years](http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000230.html),
+we want to point readers to the last release of our lessons
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: we have wanted to point learners to the latest release of our lessons

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 18]
```diff
@@ -0,0 +1,128 @@
...
+but due technical limitations on GitHub Pages
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: but due to technical limitations of GitHub Pages

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 19]
```diff
@@ -0,0 +1,128 @@
...
+and all the trouble that branches created for Git novice users
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: and the challenges of multiple branches for contributors new to Git

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 20]
```diff
@@ -0,0 +1,128 @@
...
+(for example,
+the current branch isn't clear when visiting the lesson homepage in GitHub
+and
+maintainers not be able to change the target branch of a pull request)
+we stayed with a single `gh-pages` branch in the Git repository.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: (for example, the current branch isn't obvious when you visit the lesson homepage in GitHub, and maintainers can't change the target branch of a pull request)

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 21]
```diff
@@ -0,0 +1,128 @@
...
+
+[Jonah Duckles opened a issue](https://github.com/swcarpentry/lesson-example/issues/126)
+to discuss possible solutions to this issue.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: opened a -> opened an

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 22]
```diff
@@ -0,0 +1,128 @@
...
+If you want to contribute with the discussion
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: with -> to

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 23]
```diff
@@ -0,0 +1,128 @@
...
+left your comments on the [GitHub issue](https://github.com/swcarpentry/lesson-example/issues/126).
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: left -> please leave

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 24]
```diff
@@ -0,0 +1,128 @@
...
+
+## Fully-offline-capable functionality in lesson navigation
+
+[vuw-ecs-kevin GitHub user](https://github.com/vuw-ecs-kevin)
+requested for us to improve the reader experience for those that get our lesson from [Zenodo](https://zenodo.org/),
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: requested that we improve the readers experience, if people come to our lesson from Zenodo

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 25]
```diff
@@ -0,0 +1,128 @@
...
+i.e. [from one of our releases](https://software-carpentry.org/blog/2017/08/release-2017.08.html).
+Changes on the line of [vuw-ecs-kevin's pull request](https://github.com/swcarpentry/styles/pull/166) or [Raniere's pull request](https://github.com/swcarpentry/lesson-example/pull/127)
+will be included in the next release of our lessons.
+
+## Managing workshop websites and install instructions
+
+This is another old request [[1](https://github.com/swcarpentry/DEPRECATED-bc/pull/415), [2](https://github.com/swcarpentry/DEPRECATED-bc/pull/738), [3](https://github.com/swcarpentry/workshop-template/issues/194), [4](https://github.com/swcarpentry/amy/issues/1087)].
+Edit only one line of [`index.html`](https://github.com/swcarpentry/workshop-template/blob/gh-pages/index.html)
+and have the correct instructions for the workshop.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: correct instructions -> correct setup instructions

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 26]
```diff
@@ -0,0 +1,128 @@
...
+[Jonah Duckles opened a new issue](https://github.com/swcarpentry/workshop-template/issues/421)
+to discuss ideas to archive our old request.
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: to archive -> to resolve

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 27]
```diff
@@ -0,0 +1,128 @@
...
+
+Kate Hertweck,
+Christina Koch,
+Raniere Silva
+and
+Tracy Teal
+are going to work on strategic plan document to addres this request taking in consideration the [comments on the GitHub issue](https://github.com/swcarpentry/workshop-template/issues/421).
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: plan document to addres -> plan to address taking into consideration

File: _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md [hunk 28]
```diff
@@ -0,0 +1,128 @@
...
+## Next steps
+
+We will freeze
+[lesson template](https://github.com/swcarpentry/styles)
+and
+[lesson documentation](https://github.com/swcarpentry/lesson-example)
+in October
+so maintainers have time to work on the next release.
+
+The subcommittee will meet again in November to provide update on some of the topics covered by this post
```
Comment (tracykteal) [inline _posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md]: provide update -> provide an update

**Comment by tracykteal** [comment]
Thanks @rgaiacs. Great post. I just put in comments for some wording changes. Once you let me know it's ready to go live, I'll bring that version over to the Data Carpentry blog and post there too.

**Comment by rgaiacs** [comment]
> Once you let me know it's ready to go live, I'll bring that version over to the Data Carpentry blog and post there too.

Should be good now. It has

~~~
date: 2017-09-12
time: "10:00:00"
~~~

If you decide to publish on Wednesday or later you probably want to change the information and rename the file.

**Comment by tracykteal** [comment]
LGTM. Ready to go from my perspective and it would be great if you could put it on the DC blog too.

**Comment by rgaiacs** [comment]
> it would be great if you could put it on the DC blog too

Do you want a pull request from me? I thought was easy for you to just copy and paste the file.

**Comment by tracykteal** [comment]
It is fine to copy, but it still goes in via PR. If you could put in the PR with just this file that would be great. Then I can update the YAML header to match the DC blog header.

**Comment by rgaiacs** [comment]
> It is fine to copy, but it still goes in via PR. If you could put in the PR with just this file that would be great. Then I can update the YAML header to match the DC blog header.

https://github.com/datacarpentry/datacarpentry.github.io/pull/289

**Pull request merged**